Parsedown.php

```diff
@@ -1156,9 +1138,9 @@ protected function lineElements($text, $nonNestables = array())
 
                 # cause the new element to 'inherit' our non nestables
 
-                foreach ($nonNestables as $non_nestable)
+                foreach ($nonNestables as $nonNestable)
                 {
-                    $Inline['element']['nonNestables'][] = $non_nestable;
+                    $Inline['element']['nonNestables'][] = $nonNestable;
                 }
```

Comment:
```php
$Inline['element']['nonNestables'] = array_merge($Inline['element']['nonNestables'], $nonNestable);
```

Reply:
Appending to undefined key creates it so we also have to check that the key exists with this method:
```php
$Inline['element']['nonNestables'] = isset($Inline['element']['nonNestables'])
    ? array_merge($Inline['element']['nonNestables'], $nonNestables)
    : $nonNestables
;
```

Still, I wouldn't be surprised if the behaviour currently in use gets warnings added to it at some point, and even if it doesn't, adding the check is still a good thing